src/App.tsx

```diff
@@ -1,11 +1,58 @@
-import React from 'react';
+import { Card, CardContent, makeStyles, Typography } from '@material-ui/core';
 import './App.scss';
 
-function App() {
+interface dataTS {
```

Comment:
Love the interfaces! That's really cool. The only thing, please add an id here to pass it to the `key` in the array. You can add id in a simple way by hardcoding it, or you can do it in a more proficient way, using a library like a uuid for example.

Reply:
Also it will be better (by pattern of developing) to name it `CardsDataDTO.`

Reply:
done